test/cpp/microbenchmarks/bm_chttp2_hpack.cc

```diff
@@ -753,16 +764,17 @@ class RepresentativeServerInitialMetadata {
 class RepresentativeServerTrailingMetadata {
  public:
   static std::vector<grpc_slice> GetInitSlices() {
-    return {grpc_slice_from_static_string(
-        // generated with:
-        // ```
-        // tools/codegen/core/gen_header_frame.py --compression inc --no_framing
-        // <
-        // test/cpp/microbenchmarks/representative_server_trailing_metadata.headers
-        // ```
-        "@\x0bgrpc-status\x01"
-        "0"
-        "@\x0cgrpc-message\x00")};
+    // generated with:
+    // ```
+    // tools/codegen/core/gen_header_frame.py --compression inc --no_framing
+    // --hex
+    // <
+    // test/cpp/microbenchmarks/representative_server_trailing_metadata.headers
+    // ```
+    return {MakeSlice({0x40, 0x0b, 0x67, 0x72, 0x70, 0x63, 0x2d, 0x73,
+                       0x74, 0x61, 0x74, 0x75, 0x73, 0x01, 0x30, 0x40,
+                       0x0c, 0x67, 0x72, 0x70, 0x63, 0x2d, 0x6d, 0x65,
+                       0x73, 0x73, 0x61, 0x67, 0x65, 0x00})};
```

Comment:
Side note: this has been broken for years and not doing a correct benchmark: grpc_slice_from_static_string totally stops at the trailing null byte, corrupting the stream.